src/ui/utils/thunk.ts

```diff
@@ -5,3 +5,5 @@ export interface ThunkExtraArgs {
   client: typeof clientCommands;
   ThreadFront: typeof ThreadFront;
 }
+
+export const extraThunkArgs = {} as ThunkExtraArgs;
```

Comment:
I feel like there's a story here?

Reply:
Yeah, the listener middleware has the same ability to accept an `extra` argument as the thunk middleware, and I briefly thought I'd need to access `ThreadFront.recordingId` in that listener.  

The `extraThunkArgs` object _was_ defined in `store.ts`, so I couldn't import it over to `listenerMiddleware.ts`.  So, I extracted it over to the thunk file, which is where the TS `ThunkExtraArgs` type is defined anyway, and now the listener middleware file can safely import that one object and pass it into `createListenerMiddleware({extra: extraThunkArgs})`.

(I ended up _not_ needing `ThreadFront.recordingId` after all, but hey, no reason not to have it available!)